lang/talon/talon-python-activator.talon

```diff
@@ -0,0 +1,7 @@
+app: vscode
+win.title: /— user/
+win.title: /— knausj_talon/
+win.title: / - user - Visual Studio Code/
+tag: user.python
+-
+tag(): user.talon_python
```

Comment:
tag to restrict talon-related python commands to relevant projects only. This is provided as an example for discussion.

Reply:
My suggestion would be to just leave this active for every python file by default, and either comment out or provide as an example your lines 1 through 4. That way beginning users will just have this active where necessary and more advanced users can adjust the targeting if they feel it is not working for them.